packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/momentum.scss

```diff
@@ -0,0 +1,5 @@
+$brand-font-folder: "~@momentum-ui/core/fonts";
+$icon-font-path: "~@momentum-ui/icons/fonts";
+
+// importing the base momentum so we can style the demo page
```

Comment:
We are not importing the momentum component styles here because we want the full momentum overrides for the demo.